lib/src/widgets/chat/audio/ds_audio_message_bubble.widget.dart

```diff
@@ -15,13 +15,15 @@ class DSAudioMessageBubble extends StatelessWidget {
   final String? uniqueId;
   final String audioType;
   final bool shouldAuthenticate;
+  final String? replyId;
```

Comment:
![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/c0f07186-9d0d-47cf-84ea-74d0c673c16a)

![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/88fea5b6-1eb7-4375-8243-ed3546e2bd0f)

The reply in a audio is not correct in desk app